test_package/CMakeLists.txt

```diff
@@ -1,17 +1,14 @@
 cmake_minimum_required(VERSION 2.8.11)
 project(test_package CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
 
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
-find_package(protoc CONFIG REQUIRED)
+find_package(Protobuf REQUIRED)
```

Comment:
The reason for creating `protoc` target is because we have 2 packages, protobuf and protobuf_installer. When we include Protobuf, the first one will included but not the second. So, if you need to use protoc and protobuf libraries at same project, you will need different target names.

Reply:
I understand that, in conan, protobuf and protoc are provided in 2 different packages. In CMake however, the official FindProtobuf.cmake can find both the libraries and the protoc compiler.

I did change the 'test_package/CMakeLists.txt' because the original test required to patch the upstream Protobuf cmake scripts (with protobuf.patch). I find it hard to maintain and update the patch to another protobuf version.